helm/conjur-app-deploy/charts/app-secrets-provider-p2f-jwt/templates/NOTES.txt

```diff
@@ -0,0 +1,6 @@
+The Application deployment is complete.
+The following have been deployed:
+- A sample application with a Secrets Provider init container in Push-to-File mode
+- Using authn-jwt flow
```

Comment:
Minor nit... Can we remove the `-` from this line so it's not a separate list item, i.e.:
```
- A sample application with a Secrets Provider init container in Push-to-File mode
  using authn-jwt flow
```